src/network/connection.js

```diff
@@ -317,8 +324,13 @@ module.exports = class Connection {
 
     try {
       const payloadDecoded = await response.decode(payload)
-      // KIP-219: If the response indicates that the client-side needs to throttle, do that.
+
+      /**
```

Comment:
Somewhat nit-picky, but this line wouldn't be a JSDoc-y comment, but just a multiline comment, no?

Reply:
I like the doc block because it highlights the comment. I made the change to give the importance this comment deserve; I don't think we should argue about the comment block style; my intention was just to highlight a good comment 😄